### PR TITLE
allow separate entrypoint for enums

### LINKED ISSUE #33: config to allow enums in separate entrypoint
hi - love the new enum stuff <3

one thought - would it be possible to allow a config option to let us generate them into a separate file?

defualt can stay as is - but i would like the option to say i want the runtime stuff in a separate entrypoint than the types :)

### PR DESCRIPTION
**Pull request by juliusmarminge**
closes #33

### CODE REVIEW

File: src/__test__/e2e.test.ts [hunk 1]
```diff
@@ -52,3 +52,62 @@ test(
...
+
+    const enumFile = await fs.readFile("./prisma/generated/enums.ts", {
+      encoding: "utf-8",
+    });
+    expect(enumFile).toEqual(`export type TestEnum = "A" | "B" | "C";
+export const TestEnum = {
+  A: "A",
+  B: "B",
+  C: "C",
+};
```
Comment (juliusmarminge) [inline src/__test__/e2e.test.ts:109]: separate issue but this should ideally be an `as const` object

Comment (valtyr) [reply src/__test__/e2e.test.ts:109]: Agreed!

File: src/__test__/e2e.test.ts [hunk 2]
```diff
@@ -52,3 +52,62 @@ test(
   },
   { timeout: 20000 }
 );
+
+test.only(
```
Comment (juliusmarminge) [inline src/__test__/e2e.test.ts:56]:
```suggestion
test(
```

**Review by valtyr — CHANGES_REQUESTED** [review]
Super useful addition! I have a few small comments. Otherwise feel free to add a changeset and mention the new config variable in the README, after which we can get this merged and released!

File: src/__test__/e2e.test.ts [hunk 3]
```diff
@@ -53,7 +53,7 @@ test(
   { timeout: 20000 }
 );
 
-test.only(
+test(
```
Comment (valtyr) [inline src/__test__/e2e.test.ts:56]: Haha nice save 😎

**Comment by juliusmarminge** [comment]
Made the changes you requested!

Not sure what the linter is complaining about 🤔

**Comment by valtyr** [comment]
Alrighty. I'll tame the linter... or kill it 🔪

Thanks for the contribution!

**Pull request merged**